views/sass/_components.scss

```diff
@@ -180,6 +180,21 @@ p.lead {
     }
 }
 
+.form-control,
+input[type].form-control {
```

Comment:
I'm having to overload selectors with this `input[type]` stuff because the `input[type="…"]` selectors in `_typography.scss` are just way too strong.

Bootstrap gets round this by not applying any styles to inputs by default, instead forcing you to add a `.form-control` class if you want your inputs styled. We could do the same. But I wonder whether there’s a happy medium?

Any thoughts @wrightmartin @ondenman @octopusinvitro?